docs/source/howto/aforms.rst

```diff
@@ -0,0 +1,56 @@
+.. _aforms:
+
+How to extract AcroForm fields from a PDF using Python and PDFMiner
+********************************
+
+Before you start, make sure you have :ref:`installed pdfminer.six<install>`.
+
+The second thing you need is a PDF with AcroForms (as found in PDF files with fillable forms or multiple choices).
+
+It won't work for XFA forms.
+
+.. code-block:: python
+
+    from pdfminer.pdfparser import PDFParser
+    from pdfminer.pdfdocument import PDFDocument
+    from pdfminer.pdftypes import resolve1, PDFObjRef
+    from pdfminer.psparser import PSLiteral, PSKeyword
+    from pdfminer import utils
+    
+    data = {}
+    
+    with open(file_path, 'rb') as fp:
+        parser = PDFParser(fp)
+        
+        doc = PDFDocument(parser)
+        res = resolve1(doc.catalog)
+
+        if 'AcroForm' in res:
```

Comment:
Maybe this is cleaner as e.g.
```
if 'AcroForm' not in res:
    raise ValueError("No AcroForm Found")
```
since then it's an obvious message about the problem, and we can drop an indentation level below.

What do you think?

Reply:
Will do, makes sense to me. 
(my original code had a couple of other conditions,not relevant here, so I just removed them and the it was left like that)

Reply:
Done.